docker-compose.yml

```
@@ -57,6 +57,15 @@ services:
       - prometheus
       - cadvisor
       - node-exporter
+  
+  nginx:
+    image: nginx:latest
+    volumes:
+    - <PATH>/nginx.conf:/etc/nginx/nginx.conf:rw
+    restart: unless-stopped
+    ports:
+    - 5000:80
```

What port do you suggest?

---

@MusaZwane - see below since our last port was 8082
```suggestion
    - 8083:80
```